bijection-core/src/main/scala/com/twitter/bijection/StringInjections.scala

```diff
@@ -53,7 +53,7 @@ trait StringInjections extends NumericInjections {
           val decBuf = decRef.get
           val dec = decBuf._1
           val buf = decBuf._2
-          val maxSpace = (b.length * dec.maxCharsPerByte).toInt + 1
+          val maxSpace = (b.length * (dec.maxCharsPerByte.toDouble)).toInt + 1
```

Comment:
2B = 2 billion (1 << 31), since that's how toInt handles larger values.

But yeah, Float -> Double makes more sense than integral -> Double, so that seems fine.